react/features/lobby/functions.ts

```diff
@@ -44,6 +44,18 @@ export function getKnockingParticipantsById(state: IReduxState) {
     return getKnockingParticipants(state).map(participant => participant.id);
 }
 
+/**
+ * Selector to return whether to enable the lobby chat from config.
+ *
+ * @param {IReduxState} state - State object.
+ * @returns {boolean}
+ */
+export function getEnableLobbyChat(state: IReduxState) {
+    const { enableLobbyChat, lobby } = state['features/base/config'];
+    const shouldEnableLobbyChat = enableLobbyChat || lobby?.enableLobbyChat;
```

Comment:
Instead of doing this, can you please add a translation code path in base/config/redux, like we do for similar cases? Here and elsewhere.